### PR TITLE
Stock warehouse distance

### PR DESCRIPTION
**Pull request by ddufresne**
https://www.numigi.com/web#id=17111&action=298&model=project.task&view_type=form&menu_id=200

### CODE REVIEW

File: stock_warehouse_distance/models/stock_warehouse.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, _
```
Comment (ddufresne) [inline stock_warehouse_distance/models/stock_warehouse.py]: Codacy Issue found: ['odoo.fields' imported but unused](https://app.codacy.com/app/numigi/odoo-stock-addons/pullRequest?prid=4661186)

File: stock_warehouse_distance/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from . import (
```
Comment (ddufresne) [inline stock_warehouse_distance/models/__init__.py]: Codacy Issue found: ['.stock_warehouse_distance' imported but unused](https://app.codacy.com/app/numigi/odoo-stock-addons/pullRequest?prid=4661186)

File: stock_warehouse_distance/models/stock_warehouse.py (hunk 1 above)
Comment (ddufresne) [inline stock_warehouse_distance/models/stock_warehouse.py]: Codacy Issue found: [Unused fields imported from odoo](https://app.codacy.com/app/numigi/odoo-stock-addons/pullRequest?prid=4661186)

File: stock_warehouse_distance/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+---------
+A new menu entry is added under ``Inventory / Configuration / Warehouse Distances``.
+
+.. image:: static/description/warehouse_distance_menu.png
+
+When clicking on the menu entry, the list of all distances between warehouses is displayed.
+
+.. image:: static/description/warehouse_distance_list.png
+
+Kilometers
```
Comment (foutoucour) [inline stock_warehouse_distance/README.rst]:
```suggestion
Units
```

File: stock_warehouse_distance/README.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+~~~~~~~~~~
+Distances are defined in kilometers.
+
+Reflexive
```
Comment (foutoucour) [inline stock_warehouse_distance/README.rst]:
```suggestion
Reflexivity
```

**Review by foutoucour — CHANGES_REQUESTED** [review]
2 commentaires mineures. À toi de voir si tu veux les accepter. Tu peux merger derrière.

**Pull request merged**